package.json

```diff
@@ -60,17 +60,11 @@
     "pinkie-promise": "^2.0.0"
   },
   "devDependencies": {
+    "ava": "*",
```

Comment:
I do this on all my repos. I'm ok with stuff breaking on my own repos so I can catch bugs early and prevent it to happen for my users.